core-bundle/src/Resources/contao/classes/BackendTemplate.php

```diff
@@ -64,7 +64,7 @@ public function getResponse()
 	/**
 	 * Compile the template
 	 *
-	 * @internal Do not call this method in your code. It will be made private in Contao 5.0.
+	 * @internal
 	 */
 	protected function compile()
```

Comment:
Don’t we need to make it `private` too?
```suggestion
	 * @internal
	 */
	private function compile()
```

Reply:
Changed in e9b44906d07c6ff29a56be79b873d3d0d1f8e9e5.